distributor-api-v1/operations.md

```diff
@@ -421,11 +421,11 @@ Alternative initial call used to obtain all static data about hotel relevant for
                             }
                         ]
                     }
-                },
-                "Charging": "PerRoomNight",
-                "Posting": "Daily",
-                "Ordering": 1
-            }
+                }
+            },
+            "ChargingMode": "PerTimeUnit",
+            "PostingMode": "Daily",
```

Comment:
![image](https://user-images.githubusercontent.com/6573164/126463680-b538d659-b8ec-4cc7-abfb-67bbb76d6829.png)

`PostingMode` example should be `PerTimeUnit` right?

Same thing for line 137 https://github.com/MewsSystems/gitbook-distributor-guide/pull/60/files#diff-e0d8177d43ad27101a34daf5ee1404fa5b293c8884f331abcbe852ac6671a819R137